doc/man3/DEFINE_STACK_OF.pod

```diff
@@ -16,8 +16,8 @@ OPENSSL_sk_dup, OPENSSL_sk_find, OPENSSL_sk_find_ex, OPENSSL_sk_find_all,
 OPENSSL_sk_free, OPENSSL_sk_insert, OPENSSL_sk_is_sorted, OPENSSL_sk_new,
 OPENSSL_sk_new_null, OPENSSL_sk_new_reserve, OPENSSL_sk_num, OPENSSL_sk_pop,
 OPENSSL_sk_pop_free, OPENSSL_sk_push, OPENSSL_sk_reserve, OPENSSL_sk_set,
-OPENSSL_sk_set_cmp_func, OPENSSL_sk_shift, OPENSSL_sk_sort,
-OPENSSL_sk_unshift, OPENSSL_sk_value, OPENSSL_sk_zero
+OPENSSL_sk_set_thunks, OPENSSL_sk_set_cmp_func, OPENSSL_sk_shift,
```

Comment:
No documentation for the new functions?

Reply:
I was in doubt where it should go. Other functions in this block are not documented either. I can add documentation of the type specific functions which would be consistent with the current state of the document.

Reply:
I put the docs for Lhash on this change in OPENSSL_LH_COMPFUNC.pod.  You could likely do the same for this in DEFINE_STACK_OF.pod

Reply:
> Other functions in this block are not documented either. 

That's not a valid reason.  All new public API's should have documentation. You could try running `make doc-nits` and see if it complains.

Reply:
Please have a look at the updates.